lib/services/admin.ts

```
@@ -131,20 +131,48 @@ export async function updateVendorApplicationStatus(
+      throw new Error(response.message ?? "Failed to update application status");
+    }
+
+    return response;
+  } catch (error) {
+    // Handle ApiError objects thrown by apiFetch
+    if (error && typeof error === "object" && "message" in error) {
+      const apiError = error as { message: string; status?: number };
+      const errorMessage = apiError.message ?? "Failed to update application status";
+      console.error("API Error:", {
+        message: errorMessage,
+        status: apiError.status,
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)